### ISSUE TITLE
Weird dot appears when I use JS expression on Android

### ISSUE THREAD

**Issue by napatwongchr** [GeekyAnts/NativeBase#1208]
Weird dot appears when I use JS expression in JSX on Android
It's working fine on iOS

><Footer>
          <FooterTab style={{ backgroundColor: '#FFFFFF' }}>
            <Button />
            <Button>
              <Text style={pageTextStyle}>{selectedQuestion + 1} of {newQuestions.length}</Text>
            </Button>
            <Button />
          </FooterTab>
        </Footer>

![screen shot 2560-09-05 at 4 37 45 pm](https://user-images.githubusercontent.com/12037134/30055732-8b09ef82-925a-11e7-8f93-8b80a5a40f12.png)

I've used these packages
"native-base": "^2.3.1",
"react": "16.0.0-alpha.12",
"react-native": "0.47.2"

**Comment by bansalayush** [comment]
Dots disappear if uppercase is passed as false
Try this 
```
<Footer>
          <FooterTab style={{ backgroundColor: "#FFFFFF" }}>
            <Button />
            <Button>
              <Text uppercase={false}>
                {5 + 1} of {6}
              </Text>
            </Button>
            <Button />
          </FooterTab>
        </Footer> 
```

**Comment by shivrajkumar** [comment]
@napatwongchr Fixed in version 2.3.3
